### PR TITLE
[rabbitmq][Cronus] Use prometheus plugin for rabbitmq metrics

### PR DESCRIPTION
**Pull request by s10**
bump rabbitmq dependency chart to 0.7.3

Fix for RabbitMQRPCUnackTotal alert, add standard labels

### CODE REVIEW

File: openstack/cronus/Chart.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-version: 0.1.13
+version: 0.1.14
 dependencies:
   - name: rabbitmq
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
     version: 0.6.12
   - name: rabbitmq
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.6.12
+    version: 0.7.2
```
Comment (corey-aloia) [inline openstack/cronus/Chart.yaml:11]: It looks like only 1 rabbitmq was upgraded. Is this expected?

**Review by corey-aloia — CHANGES_REQUESTED** [review]
Previously when upgrading, we came across an issue where we needed to manually exec into the pods of each environment and enable the rabbitmq features. Is this fixed? I.E. would there be any risk in this upgrade?

FWIW we use persistent volumes for rabbitmq which is why we have a different behavior in upgrades than other teams

File: openstack/cronus/Chart.yaml (hunk 1 above)
Comment (s10) [reply openstack/cronus/Chart.yaml:11]: Unexpected. Added version bump for the second one.

**Review by corey-aloia — APPROVED** [review]
Looks fine to me as long as the upgrade is low risk. I would rather @dhalimi have a look first though :)

**Comment by s10** [comment]
> Previously when upgrading, we came across an issue where we needed to manually exec into the pods of each environment and enable the rabbitmq features. Is this fixed? I.E. would there be any risk in this upgrade?
> 
> FWIW we use persistent volumes for rabbitmq which is why we have a different behavior in upgrades than other teams

This issue only affects clusters which are running version <3.12.0 and upgrading to 3.12.0+. So this enable_feature_flags was needed to run once on clusters with version 3.11.* (and, as I see, it was done).

For any future updates, we decided to enable feature flags on upgrade automatically, this is being done in helm chart rabbitmq 0.7.3+

**Review by bashar-alkhateeb — APPROVED** [review]

**Pull request merged**